tests/unit/test_runner.py

```diff
@@ -457,6 +457,7 @@ def test_random_ssh_connection_choice(
     runner._configure_runner()
     second_call_args = runner._clients.jinja.get_template("env.j2").render.call_args.kwargs
 
-    assert (
-        first_call_args["ssh_debug_info"] != second_call_args["ssh_debug_info"]
-    ), "Same ssh debug info found, this may have occurred with a very low priority."
+    assert first_call_args["ssh_debug_info"] != second_call_args["ssh_debug_info"], (
+        "Same ssh debug info found, this may have occurred with a very low priority. "
+        "Just try again."
```

Comment:
Not sure IIUC, but if the tests prints "Just try again." maybe we should remove it?

Reply:
The problem is that the test has a low probability of failing (choosing the same connection out of 100 two times in a row). I added "Just try again" so that people unfamiliar with the test don't have to dig into the test to understand why it failed.